libcaliptra/examples/generic/test.c

```diff
@@ -668,6 +668,91 @@ int rom_test_devid_csr(const test_info* info)
     return failure;
 }
 
+// Issue FW load commands repeatedly
+// Coverage for piecewise FW load and runtime FW updates
+int upload_fw_piecewise(const test_info* info)
+{
+    int failure = 0;
+    int status = boot_to_ready_for_fw(info, false);
```

Comment:
shall we sanity_check info ?

Reply:
If anything I would say to check it at run_tests() instead of each individual one. But I think either way the test is going to fail if someone breaks something.